src/tools/clippy/tests/ui/integer_arithmetic.stderr

```diff
@@ -123,6 +123,18 @@ error: integer arithmetic detected
 LL |     i >>= 2;
    |     ^^^^^^^
 
+error: integer arithmetic detected
+  --> $DIR/integer_arithmetic.rs:68:28
+   |
+LL |         const ASSOC: i32 = 1 + 1;
+   |                            ^^^^^
+
+error: integer arithmetic detected
+  --> $DIR/integer_arithmetic.rs:75:13
+   |
+LL |             1 + 1
+   |             ^^^^^
+
 error: integer arithmetic detected
   --> $DIR/integer_arithmetic.rs:82:5
    |
```

Comment:
:thinking:  this is .. interesting? It looks like the PR does not actually touch any direct clippy code?

Reply:
Those shouldn't get linted, because they are constant items. More specifically associated constants on a trait and a trait impl respectively.